commcare_export/cli.py

```diff
@@ -52,7 +52,7 @@ def add_to_parser(self, parser, **additional_kwargs):
 CLI_ARGS = [
         Argument('version', default=False, action='store_true',
                  help='Print the current version of the commcare-export tool.'),
-        Argument('query', help='JSON or Excel query file. If omitted, JSON string is read from stdin.'),
+        Argument('query', required=True, help='JSON or Excel query file. If omitted, JSON string is read from stdin.'),
```

Comment:
I think you need to drop the sentence "If omitted, JSON string is read from stdin."

Reply:
```suggestion
        Argument('query', required=True, help='JSON or Excel query file'),
```